src/clients/c++/perf_analyzer/client_backend/tensorflow_serving/CompileProto.cmake

```diff
@@ -0,0 +1,90 @@
+# A function that creates CPP sources from proto files.
+function(COMPILE_PROTO USE_GRPC PROTO_PATH OUT_PATH SRCS HDRS)
```

Comment:
I thought cmake had built-in support for generating C from proto

Reply:
I believe you mean protobuf_generate_cpp. The function produces the .pb.h and .pb.cc in the same file of invocation. But for TFS client we need the protos to retain the highly nested directory nature. I remember in  triton we would cleverly modify the import statetments in the protos to allow monolithic design but these are a lot of files here. In short, this function allows us to control the OUT_PATH of the protos and retain the nested structure from an external repository. The OUT_PATH will only carry the compiled protos without any rest of the junk from TF and TFS repos.